pkg/kube/apimirror/probe.go

```diff
@@ -70,3 +70,17 @@ type TCPSocketAction struct {
 	// +optional
 	Host string `json:"host,omitempty" protobuf:"bytes,2,opt,name=host"`
 }
+
+// GRPCAction describes an action based on GRPC health check
+type GRPCAction struct {
```

Comment:
redundant, why not use k8s directly?

Reply:
As the comment says above:
https://github.com/istio/istio/blob/1f6144eb21303a6aff9a1214eae564a81b603adb/pkg/kube/apimirror/probe.go#L15-L18